src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -686,6 +686,8 @@ export function updateDeprecatedFeaturesIntoOptions(userFeatures: DevContainerFe
 
 export async function getFeatureIdType(params: CommonParams, userFeatureId: string, lockfile: Lockfile | undefined) {
 	const { output } = params;
+	const error_message = `Legacy feature '${userFeatureId}' not supported. Please check https://containers.dev/features for replacements.
+	If you were hoping to use local Features, remember to prepend your Feature name with "./". Please check https://containers.dev/implementors/features-distribution/#addendum-locally-referenced for more information.`;
```

Comment:
```suggestion
If you were hoping to use local Features, remember to prepend your Feature name with "./". Please check https://containers.dev/implementors/features-distribution/#addendum-locally-referenced for more information.`;
```

Should we remove the tab?

Reply:
tab removed